.buildkite/pipeline.yml

```diff
@@ -1,22 +1,24 @@
+env:
+  LIVE_PIPELINE: 2023-03-24
```

Comment:
Ooh didn't know you could have env vars in Buildkite pipelines. 👀

Reply:
Copied off @paul-butcher in https://github.com/wellcomecollection/concepts-pipeline/blame/main/.buildkite/pipeline.yml!